puppet/modules/jenkins_node/manifests/init.pp

```diff
@@ -137,4 +137,22 @@
   file { '/etc/cron.daily/jenkins_cleaner':
     ensure => absent,
   }
+
+  file { '/usr/local/sbin/reboot-jenkins-node':
```

Comment:
Do you intend to use this on Debian too or should this be inside the `if` block?

Reply:
I certainly want to have a look at that, but probably not today.